tools/workspace-plugin/src/generators/split-library-in-two/generator.ts

```diff
@@ -468,13 +468,13 @@ function getImportPaths(tree: Tree, filePath: string) {
   const fileContent = tree.read(filePath, 'utf8') ?? '';
   const ast = tsquery.ast(fileContent);
 
-  const importNodes = tsquery.match(ast, 'ImportDeclaration') as import('typescript').ImportDeclaration[];
+  const importNodes = tsquery.match(ast, 'ImportDeclaration') as unknown as import('typescript').ImportDeclaration[];
```

Comment:
is `tsquery` outdated as well or mismatching tsc types ?

Reply:
hmm we have dupes, can we dedupe ? 
<img width="1079" height="317" alt="image" src="https://github.com/user-attachments/assets/c25e9337-aa3f-4f83-af1e-437a940b341a" />